### PR TITLE
guide/buses (russian)

### PR DESCRIPTION
**Pull request by ZhenyaRUS39**

### CODE REVIEW

File: docs/guide/buses/RU.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+XOD заимствует идею из электронных схем.
+
+</div>
+
+Мы можем реорганизовать патч, показанный выше, используя шины, чтобы уменьшить визуальный беспорядок:
+
+![Refactoring with buses](./with-buses.patch.png)
+
+Треугольники - это ноды-шины. Треугольник, направленный вниз - это `to-bus` , а направленный вверх 
+  `from-bus`. Обратите внимание, как мы использовали два терминала `I2C` и `ADDR` и обозначали
```
Comment (gabbapeople) [inline docs/guide/buses/RU.md]: please remove 3 spacers from the start of the line

File: docs/guide/buses/RU.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+</div>
+
+Мы можем реорганизовать патч, показанный выше, используя шины, чтобы уменьшить визуальный беспорядок:
+
+![Refactoring with buses](./with-buses.patch.png)
+
+Треугольники - это ноды-шины. Треугольник, направленный вниз - это `to-bus` , а направленный вверх 
+  `from-bus`. Обратите внимание, как мы использовали два терминала `I2C` и `ADDR` и обозначали 
+ ноды-шины. Другими словами, мы сделали две шины для нашего патча.
```
Comment (gabbapeople) [inline docs/guide/buses/RU.md]: please remove 1 spacer from the start of the line

File: docs/guide/buses/RU.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+Когда линк имеет много пересечений, замените его на шину.
+
+Когда вы возвращаете результат обратно на вход какой-либо начальной ноды, обратный линк может показаться неудобным. 
+Замена его на шину - хорошая идея.
+
+Назначая общую константу или вход как `RST`, `UPD` или `ADDR` в нескольких местах может привести к загромождению патча. 
+Замена их на шину сделает патч намного легче.
+
+С другой стороны, чрезмерное использование шин, может сделать патч сложнее для визуального отслеживания цепочки взаимодействий. 
+Так, если вы сделали три или более шины в патче, то спросите себя, может будет лучше извлечь некоторые его части в отдельные патчи.
```
Comment (gabbapeople) [inline docs/guide/buses/RU.md]: Please remove carriage return

**Comment by ZhenyaRUS39** [comment]
?

**Comment by gabbapeople** [comment]
This request is approved, you can merge it =)

**Pull request merged**